tools/enso4igv/pom.xml

```diff
@@ -91,6 +91,26 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.15</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <property name="module" value="org-netbeans-modules-textmate-lexer"/>
+                                <get src="https://repo1.maven.org/maven2/org/netbeans/api/${module}/${netbeans.version}/${module}-${netbeans.version}.nbm"
```

Comment:
- I tried [various ways](https://lists.apache.org/thread/mrdxs2q4nl7cb5ymt7m4cncdtmoqohly) to download the `.nbm` file from the Maven repository
- but at the end [old good Ant](https://ant.apache.org/) is the most straightforward solution